packages/integration/src/bitbucket/core.test.ts

```diff
@@ -97,4 +101,44 @@ describe('bitbucket core', () => {
       );
     });
   });
+
+  describe('getBitbucketDownloadUrl', () => {
+    it('add path query if a path is specified', () => {
+      const config: BitbucketIntegrationConfig  = {
+        host: 'bitbucket.mycompany.net',
+        apiBaseUrl: 'https://api.bitbucket.mycompany.net/rest/api/1.0',
+      };
+      const result = getBitbucketDownloadUrl(
+        'https://bitbucket.mycompany.net/projects/backstage/repos/mock/browse/docs',
+        config,
+      );
+      expect(result).toEqual('https://api.bitbucket.mycompany.net/rest/api/1.0/projects/backstage/repos/mock/archive?format=zip&prefix=backstage-mock&path=docs');
+    });
+
+    it('do not add path query if no path is specified', () => {
+      const config: BitbucketIntegrationConfig  = {
+        host: 'bitbucket.mycompany.net',
+        apiBaseUrl: 'https://api.bitbucket.mycompany.net/rest/api/1.0',
+      };
+      const result = getBitbucketDownloadUrl(
+        'https://bitbucket.mycompany.net/projects/backstage/repos/mock/browse',
+        config,
+      );
+      expect(new URL(result).searchParams.get('format')).toEqual('zip');
+      expect(new URL(result).searchParams.get('prefix')).not.toBeNull();
+      expect(new URL(result).searchParams.get('path')).toBeNull();
```

Comment:
I don't do assertions on the full path here because there's a bug with `git-url-parse`. 

The test would fail with:
```
Expected: "https://api.bitbucket.mycompany.net/rest/api/1.0/projects/backstage/repos/mock/archive?format=zip&prefix=mock"
Received: "https://api.bitbucket.mycompany.net/rest/api/1.0/projects/projects/backstage/repos/mock/repos/browse/archive?format=zip&prefix=browse"
```


I created a [PR](https://github.com/IonicaBizau/git-url-parse/pull/118) to fix it.